exporter/tencentcloudlogserviceexporter/example/config.yaml

```diff
@@ -1,19 +1,14 @@
 receivers:
-  prometheus:
-    config:
-      scrape_configs:
-        - job_name: 'otel-collector'
-          scrape_interval: 10s
-          static_configs:
-            - targets: ['localhost:8890']
-            - targets: ['localhost:8889']
-            - targets: ['localhost:8888']
+  otlp:
+    protocols:
+      grpc:
+        endpoint: ":4317"
```

Comment:
You don't actually need this property, as it will default to this value anyway.

Reply:
Actually, I prefer to declare the port configuration, the user may not know what the default configuration is.